pkg/controller/core/workload_controller.go

```diff
@@ -129,7 +133,31 @@ func (r *WorkloadReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	if apimeta.IsStatusConditionTrue(wl.Status.Conditions, kueue.WorkloadFinished) {
 		return ctrl.Result{}, nil
 	}
+
+	cqName, cqOk := r.queues.ClusterQueueForWorkload(&wl)
+	if cqOk {
+		// because we need to react to API cluster queue events, the list of checks from a cache can lead to race conditions
+		queueAdmissionChecks, err := r.getClusterQueueAdmissionChecks(ctx, cqName)
+		if err == nil {
+			newChecks, shouldUpdate := syncAdmissionCheckConditions(wl.Status.AdmissionChecks, queueAdmissionChecks)
+			if shouldUpdate {
+				log.V(3).Info("The workload needs admission checks updates", "clusterQueue", klog.KRef("", cqName), "queueChecks", queueAdmissionChecks)
```

Comment:
```suggestion
				log.V(3).Info("The workload needs admission checks updates", "clusterQueue", klog.KRef("", cqName), "admissionChecks", queueAdmissionChecks)
```

Reply:
the intent was to show that the list of checks are coming from the queue. 
done